counterpartylib/lib/messages/broadcast.py

```diff
@@ -186,16 +190,17 @@ def parse (db, tx, message):
         cursor.close()
         return
 
+    # stop processing if broadcast is invalid for any reason,
```

Comment:
this block doesn't even seem necessary (although it is better placed now than it was before)

maybe we should change the first `status != valid` block to just be:

`if (util.enabled('broadcast_invalid_check') or util.enabled('max_fee_fraction')) and status != 'valid':`  ??